src/openms_gui/include/OpenMS/VISUAL/APPLICATIONS/TOPPViewBase.h

```diff
@@ -476,6 +476,7 @@ protected slots:
 
     /// Main workspace
     EnhancedWorkspace ws_;  // not a pointer, but an actual object, so it gets destroyed before the DefaultParamhandler (on which it depends)
+    int activeSubwindowID_ = -1; // due to Qt bugs we need to save the current Window id in the children of the workspace;
```

Comment:
do you have a reference to a known Qt bug issue?

Reply:
no sorry. maybe it is just a documentation issue but IMHO getActiveSubwindow should always return something meaningful, even if not visible or similar. It does not.

Reply:
But now that I am thinking, it is good to have it to see if the window changed, anyway.